src/lib/components/svgView/Component.svelte

```diff
@@ -0,0 +1,62 @@
+<script lang="ts">
+	import type { iPoint } from "$lib/interfaces/iPoint";
+	import DraggableSvg from "./helpers/DraggableSVG.svelte";
+
+	export let position: iPoint = { x: 10, y: 10 };
+
+	// TODO: these should not be hard coded they should scale with the things inside the component
+	let width = 800;
+	let height = 1000;
+	const cutOff = 100;
+</script>
+
+<g transform="translate({position.x}, {position.y})">
+	<path
+		class="background stroke"
+		d="M 0 {cutOff} L {cutOff} 0 L {width} 0 L {width} {height} L 0 {height} Z"
+	/>
+	<DraggableSvg bind:position>
+		<path
+			class="bar stroke"
+			d="M {cutOff / 2} {cutOff /
+				2} L {cutOff} 0 L {width} 0 L {width} {cutOff / 2} Z"
+		/>
+		<text x={cutOff} y={cutOff / 3} font-size="2rem">Hello</text>
+		<g
+			on:pointerup={() => {
+				alert("Initiate Text Editor");
+			}}
+		>
+			<rect
+				x={width - 50}
+				y="0"
+				width={cutOff / 2}
+				height={cutOff / 2}
+				fill="pink"
+				class="stroke"
+			></rect>
+			<text x={width - 44} y="34" fill="white" font-size="28px"
+				>&lt; &gt;</text
+			>
```

Comment:
The text color should change based on the background, and the text should not overflow its container.

![image](https://github.com/ECDAR-AAU-SW-P5/Ecdar-GUI-Web/assets/20535393/1ee5977d-753e-4abd-8f9e-e86111088ad1)